inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -3,56 +3,110 @@
 import nglview as nv
 from pathlib import Path
 from typing import Literal
-from ..utils import unwrap_trajectory
+from ..utils import unwrap_trajectory, align_trajectory_to_average
```

Comment:
won't this be part of the inductiva package?

Reply:
Yes, it is. Should it not be in the utils?

Reply:
It can be, but in this way you are doing a local import, instead of a package one.